[TASK] Streamline MimeTypeValidator option handling in FileUpload

linawolf opened the pull request:
Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1149
Releases: main, 13.4

linawolf committed:
[TASK] Streamline MimeTypeValidator option handling in FileUpload

Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1149
Releases: main, 13.4

linawolf committed:
[TASK] Streamline MimeTypeValidator option handling in FileUpload

Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1149
Releases: main, 13.4

froemken commented on Documentation/ExtensionArchitecture/Extbase/Reference/Validation/Validators/_FileUploadController.php:
```diff
@@ -0,0 +1,29 @@
+                ->setUploadFolder('1:/user_upload/files/'),
+        );
+
+        $this->arguments->getArgument('myArgument')->getPropertyMappingConfiguration()->skipProperties('myPropertyName');
```
I don't understand that part. Skipping a property means: do not map that file into your model. So, in model are all other data from form, but file is missing. Yeah. Maybe it's OK, as the FileUploadConfiguration handles the file upload. BUT: Ich you store your model with an empty file, won't it overwrite the just stored file with NULL again?!?

linawolf replied:
I took the example from the changelog entry, so @garvinhicking would have to anser these questions

garvinhicking replied:
If the property is skipped, no file is uploaded. Only if everything is "there" an uploaded file will be persisted. So no overwriting here.

derhansen replied:
The same is done in `FileHandlingService->addUploadConfigurationForProperty()`. Since uploads and file reference mapping is not done via ProperyMapper, it is required to exclude file uploads fields from property mapper. 

I do not really get to part with "store your model with an empty file", because if the create action is resposible to handle and persist incoming data including file uploads, why would someone set the file field value to null (=no file) again in the same method? This would be inconsistent and result in orphaned files in the upload folder.

froemken replied:
@derhansen Hi. The default value of `myPropertyName` is declared as following:

```
protected ?FileReference $file = null;
```

So, if `myPropertyName` is skipped, the validator will be executed, the file will be uploaded, but as property mapping to `myPropertyName` is skipped the file reference identifier will not be set within the domain model. So, if you store your model, the `null` will be used and overwrites previous created file uploads. Not tested, but my understanding of that.

derhansen replied:
@froemken The mapping of file properties is performed without the extbase property mapper. That is also the reason, why property mapping in general for `FileUpload` attribute/annotatied properties is skipped in core. The `null` value will actually only be set for initially persisted objects (e.g. in a `createAction`). As soon as a file  property of an object has a value (= FileReference), modifications to the object will be applied to the object, which is fetched from persistence (e.g. Object fetched by `PersistentObjectConverter`) and thereby the object will have previously saved file references set also if property mapping for a file field is skipped.

linawolf committed:
[FEATURE] Introduce new PSR-14 BeforeLiveSearchFormIsBuiltEvent (#5474)

* [FEATURE] Introduce new PSR-14 BeforeLiveSearchFormIsBuiltEvent

Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1228
Releases: main

* Update Documentation/ApiOverview/Events/Events/Backend/BeforeLiveSearchFormIsBuiltEvent.rst

* Update Documentation/ApiOverview/Events/Events/Backend/BeforeLiveSearchFormIsBuiltEvent.rst

---------

Co-authored-by: Stefan Frömken <froemken@gmail.com>

github-actions[bot] committed:
fix: apply whitespace fixes (#5479)

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

garvinhicking committed:
[TASK] Rework CSP documentation, add reference to Config module (#5455)

* [TASK] Rework CSP documentation, add reference to Config module

Resolves #5005
Releases: main, 13.4, 12.4

* [TASK] Apply review feedback

* [TASK] Link to "Nonce"

* [TASK] Add real-world example, add sub-sections, more more more

* [TASK] Adjust for CGL

* [TASK] More CGL

* Update Index.rst

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

* Update Index.rst

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

* Update Index.rst

* Update Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update _csp_example_com.yaml

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update _csp_example_org.yaml

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update ContentSecurityPolicy.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* [TASK] Adjust intendation

* [TASK] Language checks

Releases: main

* Update _csp_example_com.yaml

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update _csp_example_org.yaml

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* [TASK] Add link to chris' CSP resources

* [TASK] Do not whitelist iframe src examples

---------

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>
Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>
Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

froemken committed:
Remove tools section from extension documentation (#5481)

* Apply CGL

* Remove tools section from extension documentation

prathers committed:
[TASK] Add FAQ item for reconnect project in Crowdin (#5484)

* [TASK] Add FAQ item for reconnect project in Crowdin

* Apply suggestions from code review

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

* Update Faq.rst

---------

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>
Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

froemken committed:
Update documentation about JS multi-step wizard (#5488)

* Update documentation about JS multi-step wizard

* Update Documentation/ApiOverview/Backend/JavaScript/Modules/MultiStepWizard.rst

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

* Move JS part into its own file

---------

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

linawolf committed:
[TASK] Update legacy installation chapter (#5503)

* Add a quick introduction
* Recommend Composer
* If no Composer recommend symlinks
* Warn about changing Core Code
* Update indentation and add anchors
* Step by step installation to be done

Releases: main, 13.4, 12.4

derhansen committed:
Added FAQ section about overriding service arguments (#5487)

* Added FAW section about overriding service arguments

Please add this section after https://review.typo3.org/c/Packages/TYPO3.CMS/+/88686 is merged to TYPO3 core.

* Update Documentation/ApiOverview/DependencyInjection/Index.rst

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

* Update Documentation/ApiOverview/DependencyInjection/Index.rst

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

* Update Documentation/ApiOverview/DependencyInjection/Index.rst

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

* Update Documentation/ApiOverview/DependencyInjection/Index.rst

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

* Update Documentation/ApiOverview/DependencyInjection/Index.rst

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

* Update Documentation/ApiOverview/DependencyInjection/Index.rst

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

---------

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

linawolf committed:
[TASK] Move Deployment chapters into a dedicated folder (#5506)

Releases: main, 13.4, 12.4

github-actions[bot] committed:
fix: apply whitespace fixes (#5511)

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

github-actions[bot] committed:
fix: apply whitespace fixes (#5513)

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

github-actions[bot] committed:
fix: apply whitespace fixes (#5510)

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

froemken committed:
Correct include dir for multi-step-wizard (#5509)

linawolf committed:
[TASK] Explain default values for Extbase properties (#5516)

* [TASK] Explain default values for Extbase properties

Releases: main, 13.4, 12.4

* [TASK] Explain default values for Extbase properties

Releases: main, 13.4, 12.4

* [TASK] Explain default values for Extbase properties

Releases: main, 13.4, 12.4

* Update Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Model.rst

---------

Co-authored-by: Stefan Frömken <froemken@gmail.com>

linawolf committed:
[TASK] Fix property examples (#5519)

* [TASK] Fix property examples

Always use at least protected for properties, private does not work

Make more clear that Blog 1 is a bad example. However I think we should change didaktics of this section in general.

Releases: main, 13.4

* Update Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_Hydrating/_Blog1.php

---------

Co-authored-by: Stefan Frömken <123929835+sfroemkenjw@users.noreply.github.com>

linawolf committed:
[TASK] Improve manual deployment guide for Composer-based installations (#5524)

* [TASK] Improve manual deployment guide for Composer-based installations

- Restructure deployment section into "Initial" and "Regular" deployment
- Add database export/import examples using mysqldump and DDEV
- Clarify which folders to transfer, exclude, or make writable
- Add external links to official CLI documentation (MySQL, DDEV)
- Use tabs for platform-specific commands
- Improve formatting and consistency with TYPO3 documentation style

Releases: main, 13.4, 12.4

* Update Documentation/Administration/Installation/Deployment/Index.rst

* Update Documentation/Administration/Installation/Deployment/Index.rst

* Update Documentation/Administration/Installation/Deployment/Index.rst

* Apply suggestions from code review

Co-authored-by: Stefan Frömken <123929835+sfroemkenjw@users.noreply.github.com>

* Update Documentation/Administration/Installation/Deployment/Index.rst

* Update Documentation/Administration/Installation/Deployment/Index.rst

---------

Co-authored-by: Stefan Frömken <123929835+sfroemkenjw@users.noreply.github.com>

masi committed:
Add example for controller argument validation (#5522)

* Add example for controller argument validation

* Update Documentation/ExtensionArchitecture/Extbase/Reference/Validation/Index.rst

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

* Update Index.rst

---------

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>
Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Move extbase domain explanations into dedicated folders (#5530)

* so that subpages can be aded
* so that codesnippets are stored close to their main page

Releases: main, 13.4, 12.4

linawolf committed:
[TASK] Move Domain chapters to subpages (#5534)

* [TASK] Move Domain chapters to subpages

Content remains unchanged

Releases: main, 13.4

* [TASK] Move Domain chapters to subpages

Content remains unchanged

Releases: main, 13.4

* [TASK] Move Domain chapters to subpages

Content remains unchanged

Releases: main, 13.4

linawolf committed:
[TASK] Typed vs Untyped properties in Extbase (#5532)

* [TASK] Typed vs Untyped properties in Extbase

Releases: main, 13.4, 12.4

* [TASK] Typed vs Untyped properties in Extbase

Releases: main, 13.4, 12.4

* Apply suggestions from code review

Co-authored-by: Stefan Bürk <stefan@buerk.tech>

---------

Co-authored-by: Stefan Bürk <stefan@buerk.tech>

linawolf committed:
[!!!][TASK] Remove "Database Relations" backend module (#5540)

References https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1239
Releases: main

linawolf committed:
[TASK] Add anchors to installation chapters (#5544)

Releases: main, 13.4, 12.4

linawolf committed:
[TASK] Rewrite Domain Overview (#5537)

* [TASK] Rewrite Domain Overview

- Reworked introduction to clarify the role of the domain layer in Extbase
- Explained real-world folder structure (Model, Repository, Validator)
- Added a very basic example

Releases: main, 13.4, 12.4

* [TASK] Rewrite Domain Overview

- Reworked introduction to clarify the role of the domain layer in Extbase
- Explained real-world folder structure (Model, Repository, Validator)
- Added a very basic example

Releases: main, 13.4, 12.4

* [TASK] Rewrite Domain Overview

- Reworked introduction to clarify the role of the domain layer in Extbase
- Explained real-world folder structure (Model, Repository, Validator)
- Added a very basic example

Releases: main, 13.4, 12.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Add new chapter for rsync-based TYPO3 deployment (#5527)

* [TASK] Add new chapter for rsync-based TYPO3 deployment

- Introduce rsync as a deployment method for Composer-based and classic projects
- Explain when to use rsync compared to zip/tar or advanced tools like Deployer
- Link to SSH and rsync documentation for beginners
- Mark placeholder for deployment command recipes (todo)

Releases: main, 13.4, 12.4

* Update Documentation/Administration/Installation/Deployment/Rsync/Index.rst

Co-authored-by: Sven Finner <63340669+sfinnerWco@users.noreply.github.com>

* Update Documentation/Administration/Installation/Deployment/Rsync/Index.rst

* Update Documentation/Administration/Installation/Deployment/Index.rst

* Update Documentation/Administration/Installation/Deployment/Index.rst

* Update Index.rst

* Update Index.rst

---------

Co-authored-by: Sven Finner <63340669+sfinnerWco@users.noreply.github.com>

linawolf committed:
[TASK] Installing TYPO3 on the server (#5551)

* [TASK] Installing TYPO3 on the server

Releases: main, 13.4, 12.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Add chapter on Version control of TYPO3 projects with Git (#5541)

* [TASK] Add chapter on Version control of TYPO3 projects with Git

Releases: main, 13.4, 12.4

* [TASK] Add chapter on Version control of TYPO3 projects with Git

Releases: main, 13.4, 12.4

* Update Documentation/Administration/VersionControl/Index.rst

* [TASK] Add chapter on Version control of TYPO3 projects with Git

Releases: main, 13.4, 12.4

---------

Co-authored-by: Stefan Frömken <123929835+sfroemkenjw@users.noreply.github.com>

linawolf committed:
[TASK] Add chapter about property types in Extbase models (#5533)

* [TASK] Add chapter about property types in Extbase models

This adds a new documentation section that explains how primitive PHP
types (string, int, float, bool) are used in Extbase models, including
typical TCA field mappings and corresponding database column types.

The chapter includes examples, validator usage, nullable handling,
and a reference table for quick lookup.

Releases: main, 13.4

* [TASK] Add chapter about property types in Extbase models

This adds a new documentation section that explains how primitive PHP
types (string, int, float, bool) are used in Extbase models, including
typical TCA field mappings and corresponding database column types.

The chapter includes examples, validator usage, nullable handling,
and a reference table for quick lookup.

Releases: main, 13.4

* [TASK] Move union types and Enums into property types

Releases: main, 13.4

* [TASK] Improve Extbase enum type docs

* Enhance with Fluid and TCA examples
* Demonstrate Localization of labels
* Use a real domain use case (scientific paper and status)

Releases: main, 13.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
Note that findByUid() and findByIdentifier() are not affected by the … (#5556)

* Note that findByUid() and findByIdentifier() are not affected by the change

Releases: main, 13.4

* Update Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Repository/Index.rst

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

* Update Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Repository/Index.rst

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Enhance installation overview (#5528)

* [TASK] Enhance installation overview

Added cards for Composer-based and classic installations.

Clarified that both methods are fully supported, and explained when which one is helpful

Releases: main, 13.4, 12.4

* [TASK] Language checks

Releases: main

* Apply suggestions from code review

Co-authored-by: Garvin Hicking <blog@garv.in>

* Update Index.rst

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>
Co-authored-by: Garvin Hicking <blog@garv.in>

masi committed:
Make clear that validators are not annotations (#5523)

* Make clear that validators are not annotations

* Update Index.rst

* Update Index.rst

---------

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

simonschaufi committed:
[BUGFIX] Fix plugin list type to CType migration (#5521)

* [BUGFIX] Fix plugin list type to CType migration

See Forge issue: https://forge.typo3.org/issues/105135

Releases: main, 13.4, 12.4

* Update Documentation/ApiOverview/ContentElements/_Migration/_typoscript.diff

Co-authored-by: Stefan Frömken <froemken@gmail.com>

* Update _typoscript.diff

---------

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>
Co-authored-by: Stefan Frömken <froemken@gmail.com>

derhansen committed:
[DOCS] Added section about extension loading order (#5512)

* [DOCS] Added section about extension loading order

* [DOCS] Added section about extension loading order

* [TASK] Language checks

Releases: main

* Update ExtensionLoadingOrder.rst

* [TASK] Incorporate suggest

Releases: main

* Update ExtensionLoadingOrder.rst

* Update ExtensionLoadingOrder.rst

* Update ExtensionLoadingOrder.rst

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>
Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

linawolf commented:
I dont know about the topic to fix this

linawolf committed:
[TASK] [BUGFIX] Respect column CHARACTER SET AND COLLATE in ext_tables.sql (#5564)

Releases: main, 13.4

linawolf committed:
[TASK] Document \Datetime as extbase model type (#5565)

* [TASK] Document \Datetime as extbase model type

Releases: main, 13.4

* [TASK] Language checks

Releases: main

* [TASK] Updated code snippet paths

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[FEATURE] Country as Extbase Type (#5567)

* [FEATURE] Country as Extbase Type

References: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1232
Releases: main

* [FEATURE] Country as Extbase Type

References: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1232
Releases: main

linawolf committed:
[TASK] Condense settings start page (#5570)

* [TASK] Condense settings start page

Releases: main, 13.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Merge and condense config overview chapters (#5569)

* [TASK] Merge and condense config overview chapters

As a preparation for including the
site sets here

Releases: main, 13.4, 12.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

achimfritz committed:
Update SYS.rst (#5572)

linawolf committed:
[TASK] Database overview page (#5568)

* [TASK] Database overview page

Moving dbal in its own chapter so that other
DB topics can also be introduced

Releases: main, 13.5

* [TASK] Database overview page

Moving dbal in its own chapter so that other
DB topics can also be introduced

Releases: main, 13.5

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

franzholz committed:
site sets small changes (#5577)

* small changes - part 1

* more modifications

* Update Documentation/ApiOverview/SiteHandling/SiteSets.rst

---------

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

linawolf committed:
[TASK] Update database tables section (#5579)

* [TASK] Update database tables section

Most managed tables are now defined by TCA

Releases: main, 13.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

github-actions[bot] committed:
fix: apply whitespace fixes (#5583)

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

linawolf committed:
[TASK] Update database upgrade section (#5580)

* [TASK] Update database upgrade section

Most managed tables are now defined by TCA

Releases: main, 13.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

georg-tiefenbrunn committed:
Fix inlineLocalizeSynchronize code example (#5584)

Add missing colon/semicolon to ensure valid array definition

linawolf committed:
[TASK] Mention Extbase Queries in Database section (#5582)

* [TASK] Mention Extbase Queries in Database section

Releases: main, 13.4

* [TASK] Mention Extbase Queries in Database section

Releases: main, 13.4

* Update Documentation/ApiOverview/Database/Extbase/Index.rst

---------

Co-authored-by: Stefan Frömken <froemken@gmail.com>

linawolf committed:
[TASK] Unify calling non-Composer "Classic mode" (#5589)

Releases: main, 13.4, 12.4

PascaleBeier committed:
[TASK] Fix links in Upgrade Third-Party-Tools (#5590)

linawolf committed:
[TASK] Update Deployer Section (#5591)

* [TASK] Update Deployer Section

Releases: main, 13.4, 12.4

* [TASK] Update Deployer Section

Releases: main, 13.4, 12.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

SplitterAlex committed:
Fix typo in Model.rst (#5595)

remove duplicate "the"

SplitterAlex committed:
Update ModuleConfiguration.rst (#5594)

Fix typo

linawolf committed:
[TASK] Move Deployment and Tuning out of Installation into Administra… (#5593)

* [TASK] Move Deployment and Tuning out of Installation into Administration

They are not really related to installing TYPO3

Releases: main, 13.4, 12.4

* [TASK] Move Deployment and Tuning out of Installation into Administration

They are not really related to installing TYPO3

Releases: main, 13.4, 12.4

pschriner committed:
`NEW` must nocht be followed by an underscore (#5598)

* [TASK] Clarify datahandler new uid requirements

* Update Documentation/ApiOverview/DataHandler/Database/Index.rst

* Update Index.rst

---------

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

linawolf committed:
[TASK] Remove Chapters on TYPO3 Surf and Magephp (#5605)

* [TASK] Remove Chapters on TYPO3 Surf and Magephp

They are rarely used nowadays and the descriptions outdated

Releases:  main, 13.4, 12.4

* [TASK] Remove Chapters on TYPO3 Surf and Magephp

They are rarely used nowadays and the descriptions outdated

Releases:  main, 13.4, 12.4

linawolf committed:
[TASK] CI/CD in deployment for TYPO3 projects (#5601)

* [TASK] CI/CD in deployment for TYPO3 projects

Resolves: https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues/5499
Releases:  main, 13.4, 12.4

* Update Documentation/Administration/Deployment/Automated/Index.rst

* Update Documentation/Administration/Deployment/Automated/Index.rst

* Update Documentation/Administration/Deployment/Automated/Index.rst

* Update Index.rst

* Update Documentation/Administration/Deployment/Automated/Index.rst

* Update Documentation/Administration/Deployment/Automated/Index.rst

* Update Index.rst

* Update Index.rst

---------

Co-authored-by: Stefan Frömken <123929835+sfroemkenjw@users.noreply.github.com>

github-actions[bot] committed:
fix: apply whitespace fixes (#5610)

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

linawolf committed:
[TASK] How to access the request, cobject and DB in a ViewHelper (#5604)

* [TASK] How to access the request, cobject and DB in a ViewHelper

Releases: main, 13.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Update and improve Classic mode installation chapters  (#5612)

* [TASK] Update and improve Classic mode installation chapters

Releases:  main, 13.4

* [TASK] Update and improve Classic mode installation chapters

Releases:  main, 13.4

* Update Documentation/Administration/Installation/ClassicMode/WgetSymlink.rst

* Update Documentation/Administration/Installation/ClassicMode/TarballZip.rst

* Update Documentation/Administration/Installation/ClassicMode/WgetSymlink.rst

* Update Documentation/Administration/Installation/ClassicMode/WgetSymlink.rst

* Update Documentation/Administration/Installation/ClassicMode/Windows.rst

* Update TarballZip.rst

* [TASK] Update and improve Classic mode installation chapters

Releases:  main, 13.4

brotkrueml committed:
adjust note that TSFE->no_cache is not available on v13.4 (#5615)

The note should reflect this.

Releases: main, 13.4

linawolf committed:
[TASK] Deployment running Git pull and Composer (#5614)

* [TASK] Deployment running Git pull and Composer

Releases:  main, 13.4, 12.4

* [TASK] Deployment running Git pull and Composer

Releases:  main, 13.4, 12.4

* [TASK] Deployment running Git pull and Composer

Releases:  main, 13.4, 12.4

* [TASK] Language checks

Releases: main

* Update Index.rst

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

github-actions[bot] committed:
fix: apply whitespace fixes (#5623)

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

linawolf committed:
[TASK] Update Security Guidelines for administrators (#5625)

* [TASK] Update Security Guidelines for administrators

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Update Restrict access Security Guidelines (#5628)

* [TASK] Update Restrict access Security Guidelines

Releases: main, 13.4. 12.4

* [TASK] Update Restrict access Security Guidelines

Releases: main, 13.4. 12.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Update File integrity Security Guidelines (#5631)

* [TASK] Update File integrity Security Guidelines

Releases: main, 13.4. 12.4

* [TASK] Update File integrity Security Guidelines

Releases: main, 13.4. 12.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Update main security chapter titles, navigation, and introductions (#5640)

* [TASK] Update main chapter titles, navigation, and introductions

- Revised main section titles and navigation titles for clarity
- Improved introductory text for better context and structure
- Corrected outdated indentation for consistency
- Note: Detailed content review and overhaul of individual chapters is still pending

Releases: main, 13.4. 12.4

* [TASK] Update main chapter titles, navigation, and introductions

- Revised main section titles and navigation titles for clarity
- Improved introductory text for better context and structure
- Corrected outdated indentation for consistency
- Note: Detailed content review and overhaul of individual chapters is still pending

Releases: main, 13.4. 12.4

linawolf committed:
Fix display errors (#5643)

Releases: main, 13.4, 12.4

linawolf committed:
[BUGFIX] Align Extbase DateTime handling to FormEngine and DataHandler (#5645)

Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1253
Releases: main, 13.4

linawolf committed:
[TASK] Move Backup into Administration chapter (#5639)

* [TASK] Move Backup into Administration chapter

And create an additional Backup Security chapter

Releases: main, 13.4. 12.4

* [TASK] Move Backup into Administration chapter

And create an additional Backup Security chapter

Releases: main, 13.4. 12.4

* [TASK] Move Backup into Administration chapter

And create an additional Backup Security chapter

Releases: main, 13.4. 12.4

* Update Documentation/Security/GuidelinesAdministrators/Backups.rst

* Update Documentation/Security/GuidelinesAdministrators/Backups.rst

* Update Documentation/Security/GuidelinesAdministrators/Backups.rst

---------

Co-authored-by: Stefan Frömken <froemken@gmail.com>

linawolf committed:
[TASK] Update HTTPS chapter (#5636)

Releases: main, 13.4. 12.4

linawolf committed:
[TASK] Running TYPO3 classic mode demo using Docker (#5616)

* [TASK] Running TYPO3 classic mode demo using Docker

Releases:  main, 13.4, 12.4

* Update Documentation/Administration/Installation/ClassicMode/Docker.rst

* Update Documentation/Administration/Installation/ClassicMode/Docker.rst

* Update Documentation/Administration/Installation/ClassicMode/Docker.rst

* [TASK] Why docker only

* [TASK] Docker and Docker Compose

* Update Documentation/Administration/Docker/DockerDemo/_codesnippets/_DockerDbDemo.sh

* Update _docker-compose.yaml

* Update Documentation/Administration/Docker/DockerComposeDemo/_codesnippets/_docker-compose.yaml

Co-authored-by: Jaap van Otterdijk <jaap@ijaap.nl>

---------

Co-authored-by: Jaap van Otterdijk <jaap@ijaap.nl>

linawolf committed:
[TASK] Multi-stage environment workflows (#5622)

* [TASK] Multi-stage environment workflows

Releases:  main, 13.4, 12.4

* [TASK] Multi-stage environment workflows

Releases:  main, 13.4, 12.4

* [TASK] Multi-stage environment workflows

Releases:  main, 13.4, 12.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Extension installation with composer (#5653)

* [TASK] Extension installation with composer

Releases: main, 13.4, 12.4

* [TASK] Extension installation with composer

Releases: main, 13.4, 12.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Improve overview Cards in the Admin chapter (#5652)

Releases: main, 13.4, 12.4

linawolf committed:
Add page stub about security to Administration (#5651)

making related topics easier to find

linawolf committed:
[SECURITY] Enforce file extension and MIME-type consistency (#5644)

* [SECURITY] Enforce file extension and MIME-type consistency

Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1252
Releases: main, 13.4. 12.4

* Update Documentation/Configuration/Typo3ConfVars/SYS.rst

Co-authored-by: Stefan Frömken <froemken@gmail.com>

---------

Co-authored-by: Stefan Frömken <froemken@gmail.com>

linawolf committed:
[TASK] Fix rendering errors (#5663)

Already fixed for 12.4

Releases: main, 13.4

garvinhicking committed:
Update MigrationListType.rst (#5667)

Fixes #5666

linawolf committed:
[TASK] Update Database security chapter (#5638)

* [TASK] Update Database security chapter

Releases: main, 13.4. 12.4

* Update Documentation/Security/GuidelinesAdministrators/DatabaseAccess.rst

---------

Co-authored-by: Stefan Frömken <froemken@gmail.com>

linawolf committed:
[TASK] Update Directory Indexing chapter (#5637)

* [TASK] Update Directory Indexing chapter

Releases: main, 13.4. 12.4

* Update Documentation/Security/GuidelinesAdministrators/DirectoryIndexing.rst

* Korrekt indentation

---------

Co-authored-by: Stefan Frömken <froemken@gmail.com>

linawolf committed:
[TASK] Clarify File permissions vs HTTP access (#5635)

* [TASK] Clarify File permissions vs HTTP access

And overhaul File permissions chapter

Releases: main, 13.4. 12.4

* Update Documentation/Security/GuidelinesAdministrators/FileDirectoryPermissions.rst

* Update Documentation/Security/GuidelinesAdministrators/FileDirectoryPermissions.rst

---------

Co-authored-by: Stefan Frömken <froemken@gmail.com>

linawolf committed:
[TASK] Update more admin Security Guidelines (#5634)

* [TASK] Update more admin Security Guidelines

Releases: main, 13.4. 12.4

* [TASK] Update more admin Security Guidelines

Releases: main, 13.4. 12.4

julianhofmann committed:
[BUGFIX] Annotate correct variable (#5679)

`$nonce` is getting a string returned by `$nonceAttribute->consume()`
The annotated ConsumableString is return-type of  `$this->getRequest()->getAttribute('nonce')` and thus belonging to `$nonceAttribute`

linawolf committed:
[TASK] Demonstrate how to extend a docker image (#5659)

* [TASK] Demonstrate how to extend a docker image

Releases: main, 13.4, 12.4

* [TASK] Demonstrate how to extend a docker image

Releases: main, 13.4, 12.4

* Update Documentation/Administration/Docker/AutomateSetup/Index.rst

Co-authored-by: Sven Finner <63340669+sfinnerWco@users.noreply.github.com>

* Update Documentation/Administration/Docker/AutomateSetup/Index.rst

Co-authored-by: Sven Finner <63340669+sfinnerWco@users.noreply.github.com>

* Update Documentation/Administration/Docker/AutomateSetup/Index.rst

---------

Co-authored-by: Sven Finner <63340669+sfinnerWco@users.noreply.github.com>

linawolf committed:
[TASK] Docker Deployment and Distribution (#5683)

* [TASK] Docker Deployment and Introduction

Releases: main, 13.4, 12.4

* [TASK] Docker Container and Database

Releases: main, 13.4, 12.4

* [TASK] Mention Gitlab Container Registry

Releases: main, 13.4, 12.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Update system requirement page (#5669)

Releases: main, 13.4, 12.4

koehnlein committed:
Update f:layout in extbase module tutorial (#5686)

* Update f:layout in extbase module tutorial

* Fix typo

* Update Documentation/ExtensionArchitecture/HowTo/BackendModule/CreateModuleWithExtbase.rst

---------

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

linawolf committed:
[TASK] Dealing with file permissions in docker (#5680)

* [TASK] Dealing with file permissions in docker

Releases: main, 13.4, 12.4

* [TASK] Language checks

Releases: main

* [TASK] Docker on production

Releases: main, 13.4, 12.4

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Collect production settings in one chapter (#5695)

Releases: main, 13.4, 12.4

linawolf committed:
[TASK] Deployment with Rsync (#5698)

* [TASK] Deployment with Rsync

Releases: main, 13.4, 12.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Rsync Deployment improvements (#5701)

Releases: main, 13.4, 12.4

linawolf committed:
[TASK] Running TYPO3 behind a reverse proxy (#5697)

* [TASK] Running TYPO3 behind a reverse proxy

Releases: main, 13.4, 12.4

* [TASK] Running TYPO3 behind a reverse proxy

Releases: main, 13.4, 12.4

garvinhicking committed:
[TASK] Use mergeRecursiveWithOverrule in PSR site config event (#5704)

See https://forge.typo3.org/issues/106813 - the array_merge_recursive()
function has the drawkback of introducing duplicate keys.

Instead, this now prefery mergeRecursiveWithOverrule, which can also
handle the '__UNSET' special values.

linawolf committed:
[TASK] Reverse proxies on containers (#5706)

Releases: main, 13.4, 12.4

froemken committed:
Create entry in log file (#5710)

...and update indents

linawolf committed:
[TASK] Link to docker-composer from pure docker setup (#5709)

froemken committed:
Update Index.rst (#5715)

Just replace colon with comma

froemken committed:
Update explanation for docker start/stop (#5718)

* Update explanation for docker start/stop

* Use just docker start/stop

github-actions[bot] committed:
fix: apply whitespace fixes (#5721)

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

linawolf committed:
[TASK] Mention how to upload additional.php without vim (#5723)

* Mention how to upload additional.php without vim

Releases: main, 13.4, 12.4

* Apply suggestions from code review

---------

Co-authored-by: Stefan Frömken <froemken@gmail.com>

linawolf committed:
Remove || true from manual bash script (#5722)

Releases: main, 13.4, 12.4

franzholz committed:
plugin registration in TCA/Overrides (#5728)

Not directly in TCA!

linawolf committed:
[TASK] Automatically build and tag Docker images in CI/CD pipelines (#5731)

* [TASK] Automatically build and tag Docker images in CI/CD pipelines

Releases: main, 13.4, 12.4

* [TASK] Automatically build and tag Docker images in CI/CD pipelines

Releases: main, 13.4, 12.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

brotkrueml committed:
Fix example for SiteConfigurationLoadedEvent (#5736)

linawolf committed:
Fix indentation (#5737)

Releases main, 13.4, 12.4

linawolf committed:
[TASK] Improve directory structure chapters (#5732)

* [TASK] Improve directory structure chapters

Explain how to detect Composer mode vs classic mode and link between the two

Improve contents menus and headlines

Releases: main, 13.4, 12.4

* [TASK] Improve directory structure chapters

Explain how to detect Composer mode vs classic mode and link between the two

Improve contents menus and headlines

Releases: main, 13.4, 12.4

* Update Documentation/Administration/DirectoryStructure/ClassicInstallations.rst

Co-authored-by: Stefan Frömken <123929835+sfroemkenjw@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Stefan Frömken <123929835+sfroemkenjw@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Stefan Frömken <123929835+sfroemkenjw@users.noreply.github.com>

* Apply suggestions from code review

---------

Co-authored-by: Stefan Frömken <123929835+sfroemkenjw@users.noreply.github.com>

github-actions[bot] committed:
fix: apply whitespace fixes (#5743)

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

mkoitka committed:
Fix caption of deploy.yaml include (#5751)

froemken committed:
Update event listener example #3155 (#5750)

froemken committed:
Correct extbase registerPlugin #5431 (#5749)

froemken committed:
[FEATURE] Add events for getObjectCountByQuery 1241 (#5747)

* Update MessageBus. ChangeLog2docs 1245

* A two new events. ChangeLog2docs 1241

* Update Documentation/ApiOverview/Events/Events/Extbase/Persistence/ModifyQueryBeforeFetchingObjectCountEvent.rst

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

* Update Documentation/ApiOverview/Events/Events/Extbase/Persistence/ModifyQueryBeforeFetchingObjectCountEvent.rst

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

* Update Documentation/ApiOverview/MessageBus/Index.rst

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

* Add header links

* Add missing header anchors

---------

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>
Co-authored-by: Stefan Froemken <sfroemken@jweiland.net>

linawolf committed:
[TASK] Bugfix add missing files (#5760)

Follow up for https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/pull/5747

Releases: main

linawolf committed:
[TASK] Update code snippets (#5761)

Releases: main

froemken committed:
CleanUp AuthenticationService (#5762)

froemken committed:
Add X-Forwarded-Proto. ChangeLog2docs 1214 (#5748)

froemken committed:
[BUGFIX] Allow to configure RateLimiters in message consumer 1245 (#5746)

* Update MessageBus. ChangeLog2docs 1245

* Update Documentation/ApiOverview/MessageBus/Index.rst

---------

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

linawolf committed:
[TASK] Add production logging configuration guide (#5758)

* [TASK] Add production logging configuration guide

Add new chapter on configuring TYPO3 logging for production environments

Covers log levels, rotation, deprecation log disabling, centralized error
monitoring (e.g., Sentry), and system log management tools

Clarify separation between developer-focused logging API and production usage;
add cross-links between related chapters for better navigation

Backport to 12.4 would have to be done manually due to the introduction of the rotatinglogwriter with v13.0

Releases: main, 13.4

* [TASK] Add production logging configuration guide

Add new chapter on configuring TYPO3 logging for production environments

Covers log levels, rotation, deprecation log disabling, centralized error
monitoring (e.g., Sentry), and system log management tools

Clarify separation between developer-focused logging API and production usage;
add cross-links between related chapters for better navigation

Backport to 12.4 would have to be done manually due to the introduction of the rotatinglogwriter with v13.0

Releases: main, 13.4

* [TASK] Language checks

Releases: main

* Update Documentation/Administration/Production/Logging/Index.rst

* Apply suggestions from code review

* [TASK] Update code snippets

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

mkiebele committed:
Feature/add missing link in wget installation (#5778)

* [TASK] Add hint about installation wizard in wget installation

* [TASK] Add hint about FIRST_INSTALL

* [TASK] Adjust anchor

---------

Co-authored-by: Kiebele <mkiebele@w-commerce.de>

linawolf committed:
[TASK] Update chapter major update (#5777)

* [TASK] Update chapter major update

* Introduce permalinks where missing
* Reorder section to make a more logic flow
* Remove sections that do not make sense anymore
* Add tipps about first updating PHP and all available extensions

Releases: main, 13.4, 12.4

* [TASK] Language checks

Releases: main

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Add port mapping and reverse proxy guidance (#5744)

* [TASK] Add port mapping and reverse proxy guidance

Add new chapter on port usage in container-based TYPO3 hosting

Explain how internal port 80 maps to external ports and when reverse proxies
are required for HTTPS

Point users to reverse proxy chapter and highlight site config correction if
TYPO3 generates an incorrect base URL

Releases: main, 13.4, 12.4

* [TASK] Add port mapping and reverse proxy guidance

Add new chapter on port usage in container-based TYPO3 hosting

Explain how internal port 80 maps to external ports and when reverse proxies
are required for HTTPS

Point users to reverse proxy chapter and highlight site config correction if
TYPO3 generates an incorrect base URL

Releases: main, 13.4, 12.4

* Apply suggestions from code review

* Update Documentation/Administration/Docker/Production/Ports/Index.rst

* Update Documentation/Administration/Docker/DockerComposeDemo/Index.rst

* Update Documentation/Administration/Docker/Production/Ports/Index.rst

* Update Documentation/Administration/Docker/Production/Ports/Index.rst

---------

Co-authored-by: Stefan Frömken <froemken@gmail.com>

sarahmccarthy123 committed:
[TASK] Add link to the Github Action Template inside the Deployer documentation (#5780)

#5772

Releases: main, 13.4, 12.4

linawolf committed:
Create the folder for the document root if it does not exist (#5786)

Releases: main, 13.4, 12.4

mkiebele committed:
Hint database credentials (#5788)

* [TASK] Adds where the credentials for the database can be entered.

* [TASK] Adds where the credentials for the database can be entered.

* [TASK] Adds where the credentials for the database can be entered.

* Update Documentation/Administration/Deployment/Index.rst

Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

---------

Co-authored-by: Kiebele <mkiebele@w-commerce.de>
Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>
Co-authored-by: Chris Müller <2566282+brotkrueml@users.noreply.github.com>

github-actions[bot] committed:
fix: apply whitespace fixes (#5791)

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

linawolf committed:
[TASK] Pre and Post Updgrade steps from includes (#5793)

As a first step for clean up, move the pre and post upgrade
steps from the includes into the positions they are actually used.

A general cleanup will be done in follow ups.

Releases: main, 13.4, 12.4

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

mkiebele committed:
Link to deployerrsync from GitLab template (#5792)

* [TASK] Add link to deployerRsync from GitLab template

* [TASK] Add link to deployerRsync from GitLab template

---------

Co-authored-by: Kiebele <mkiebele@w-commerce.de>

linawolf committed:
[TASK] Bugfix link backup instead of including (#5796)

Already fixed in 12.4

Releases: main, 13.4

linawolf committed:
[BUGFIX] Remove duplicated sections on reference index (#5801)

Releases: main, 13.4, 12.4

sarahmccarthy123 committed:
[TASK] Re-arrange deployment docs to separate out initial and incremental deployment and add links (#5800)

* [TASK] Re-arrange deployment docs to separate out initial and incremental deployment and add links

Releases: main, 13.4, 12.4

* [TASK] Re-arrange deployment docs to separate out initial and incremental deployment and add links

Releases: main, 13.4, 12.4

mwanush committed:
Update SiteSettingDefinitions.rst (#5798)

linawolf committed:
[TASK] List Admin Tools in Admin Section (#5802)

Releases: main, 13.4, 12.4

linawolf committed:
[BUGFIX] Remove reference to version wizard / combat mode (#5810)

It has been removed with https://docs.typo3.org/permalink/changelog:breaking-24900 in TYPO3 7.1!

Releases: main, 13.4, 12.4

linawolf committed:
[TASK] List modules in Configuration chapter (#5775)

* [TASK] List modules in Configuration chapter

Extending the Configuration module has been
moved to EXT:lowlevel docs with https://review.typo3.org/c/Packages/TYPO3.CMS/+/89782

Releases: main, 13.4

* [TASK] Language checks

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

github-actions[bot] committed:
fix: apply whitespace fixes (#5815)

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

david-bonhagen committed:
[TASK] Add relevant links to deployer recipe and ci config (#5816)

linawolf committed:
[TASK] Explain Admin Tools > Maintenance in dedicated chapter (#5814)

* [TASK] Explain Admin Tools > Maintenance in dedicated chapter

And only link to it from the post upgrade taks

Releases: main, 13.4, 12.4

* [TASK] Language checks

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
Warn that method must be called initializeObject (#5822)

* Warn that method must be called initializeObject

Releases: main, 13.4, 12.4

* [TASK] Language checks

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

linawolf committed:
[TASK] Explain details on Environment module (#5819)

* [TASK] Explain details on Environment module

Releases: main, 13.4, 12.4

* [TASK] Language checks

---------

Co-authored-by: Sarah McCarthy <sarahmccarthy123@yahoo.com>

david-bonhagen committed:
[FIX] Broken link to local edit docs (#5818)

* [FIX] Broken link to local edit docs

* Update CONTRIBUTING.md

Use permalinks

---------

Co-authored-by: Lina Wolf <48202465+linawolf@users.noreply.github.com>

linawolf committed:
Add anchors to ExtTables.rst (#5829)

Releases: main, 13.4, 12.4

github-actions[bot] committed:
fix: apply whitespace fixes (#5832)

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

garvinhicking commented:
Creating a follow-up. Sorry for the GIT mess I made when I tried to update this patch.
